### PR TITLE
Add sniper version 1.0.0

### PR DESCRIPTION
**Pull request by toshi0383**
Hello!

I have pre-built binary in `bin/sniper`.
Please tell me how to express that in sniper.rb Formula.

### CODE REVIEW

**Comment by dunn** [comment]
It needs to be build from source if possible, otherwise you'll have to submit it to our [binary](https://github.com/Homebrew/homebrew-binary) tap.

File: Library/Formula/sniper.rb [hunk 1]
```diff
@@ -0,0 +1,7 @@
+class Sniper < Formula
+  desc "osx command line app terminator"
+  homepage "https://github.com/toshi0383/sniper"
+  url "https://github.com/toshi0383/sniper.git", :tag => "1.0.0", :revision => "8a655740145b0f9846b02eadfd491b08ef287e57"
+  head "https://github.com/toshi0383/sniper.git"
+
```
Comment (dunn) [inline Library/Formula/sniper.rb]: You're missing an install and test block; take a look at the Formula Cookbook: https://github.com/Homebrew/homebrew/blob/master/share/doc/homebrew/Formula-Cookbook.md

Comment (toshi0383) [inline Library/Formula/sniper.rb]: In which section should I look up ?

Comment (mikemcquaid) [inline Library/Formula/sniper.rb]: @toshi0383 Read it all.

**Comment by toshi0383** [comment]
```
homebrew $ ./bin/brew audit --strict --online sniper
==> brew style sniper

1 file inspected, no offenses detected

==> audit problems
sniper:
 * GitHub repository not notable enough (<20 forks, <20 watchers and <50 stars)

Error: 1 problem in 1 formula
```

Looks like sniper is not notable enough for Homebrew yet.
Closing this for now. Will open again when it becomes😉

**Pull request closed without merging**